modules/release-notes/ref-release-notes-developer-preview.adoc

```diff
@@ -14,7 +12,22 @@ Developer Preview features are not supported by Red Hat in any way and are not f
 For more information about the support scope of Red Hat Developer Preview features, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
 ====
 
-== {ls-brand-name}
+[id="developer-preview-rhidp-8709"]
+== Configurable Key Performance Indicators (KPIs) by using the Scorecard plugin
+
+With this release, {product} integrates the Scorecard plugin, enabling users to visualize software quality and health metrics directly in {product-short}.
+
+Developers can visualize scorecards with the metrics available to the their role in a tab on component detail pages, allowing them to quickly check the quality of their code and ensure it meets team standards.
+
+Platform engineers can configure Key Performance Indicators (KPIs) from various data sources, with initial support for GitHub open pull requests and Jira open issues. Custom metric providers will allow for the addition of more data sources.
+
+This simplifies the monitoring of applications by offering a single, clear report on health, security, and compliance.
+
+// RN already in https://issues.redhat.com/browse/RHIDP-8709
+
+
+.Additional resources
+* link:https://issues.redhat.com/browse/RHIDP-8709[RHIDP-8709]
+
+
 
```

Comment:
Also in developer preview are:

Lightspeed (now based on Llama Stack) @karthikjeeyar 
MCP tools for Backstage (@johnmcollier )
Openshift AI Connector for RHDH (@gabemontero )

Reply:
Also in Developer preview is the RHDH Dynamic Plugins Factory work by @Zaperex 
Customers often struggle with converting static plugins to dynamic plugins, so having a link from the release notes to this new repo is very important.